### PR TITLE
added tests for fflib_QualifiedMethodAndArgValues

### PR DESCRIPTION
**Pull request by evanmcd**
This will address issue #74

### CODE REVIEW

File: sfdx-source/apex-mocks/main/default/classes/fflib_QualifiedMethodAndArgValuesTest.cls [hunk 1]
```diff
@@ -0,0 +1,24 @@
+@isTest
+public with sharing class fflib_QualifiedMethodAndArgValuesTest
```
Comment (stohn777) [inline sfdx-source/apex-mocks/main/default/classes/fflib_QualifiedMethodAndArgValuesTest.cls]: `private class ...`?

Comment (ImJohnMDaniel) [reply sfdx-source/apex-mocks/main/default/classes/fflib_QualifiedMethodAndArgValuesTest.cls]: I agree @stohn777.  There is no need to have `with sharing` in a test class

Comment (evanmcd) [reply sfdx-source/apex-mocks/main/default/classes/fflib_QualifiedMethodAndArgValuesTest.cls]: OK, yeah agreed. I was just going off of other tests in the codebase that had it (https://github.com/apex-enterprise-patterns/fflib-apex-common/blob/master/fflib/src/classes/fflib_SObjectDomainTest.cls, for example).

**Review by ImJohnMDaniel — APPROVED** [review]

**Review by stohn777 — APPROVED** [review]

**Comment by daveespo** [comment]
@evanmcd  -- thanks for the PR!

**Pull request merged**